obs/preinstallimage-bios.sh.in

```diff
@@ -252,6 +252,15 @@ sed -e 's|.*RuntimeMaxFileSize.*|RuntimeMaxFileSize=10M|' \
     -e 's|.*SyncIntervalSec.*|SyncIntervalSec=1s|' \
     -i /etc/systemd/journald.conf
 
+## journald persistency setup (SyncIntervalSec enlarged)
+## see below 'MaxLevelStore' non-devel setting
+
+sed -e 's|.*Storage.*|Storage=persistent|' \
+    -e 's|.*SystemMaxUse.*|SystemMaxUse=300M|' \
+    -e 's|.*SystemMaxFileSize.*|SystemMaxFileSize=10M|' \
+    -e 's|.*SyncIntervalSec.*|SyncIntervalSec=2s|' \
+    -i /etc/systemd/journald.conf
```

Comment:
This seems to re-edit the same file we have just edited a command above. I believe this highlights a need for different (deployment-dependent) settings; probably the small volatile (in-RAM) journal made more sense on ARM IPCs where IOPS are low.

For OVAs this would depend on backing store of the VM farm really, but we may assume something sufficiently fast. Faster than original IPCs' flash at least.

So this part of PR asks for an `if`.

Reply:
Just in case for persistence, note that we have `rsyslogd` picking up data from journald and storing it locally and/or forwarding over syslog to a remote sinks, if enabled via config and according to predefined level of detail (see its config files and `loghost-rsyslog` setup script; I don't think we have Web-UI for tickling it yet). For example, our lab uses it extensively to have gigabytes and many months worth of debug logs stored on a server off the recyclable HW or virtual RCs, and this allows for a lot of data mining to analyze the fun part of "sporadic" issues.

So maybe we do not really need another copy in `journald` persisted logs, but another setting (or another default) for rsyslog config?

Reply:
Yes. i reedit the journald.conf file to keep the previous 'volatile' config as is.
What is the criteria to distinguish IPC (low) vs OVA (stronger) context?
We redirect our logs to syslog with a log4cplus SysLog appender... and use systemd-journald to persist these logs. journalctl seems to be a powerful tool to extract log messages from journald logfiles, with a lot of options, filters, output formats, etc... Finally, the 'first step' target is to provide a set of logfiles (one full logs archive) covering the activity of the system (kernel, boot, ...), the native & java part of IPM2, and our REST api IO (for post,put,delete) . Handling remote logs w/ rsyslog is not (yet) in the scope.

Reply:
Note that this "handling remote syslogs" is already in code and practice for several years however ;)

As for IPC vs OVA, maybe the not so clean but practical solution can be to look at the CPU arch of the builder (this would however cause IPC-like X86 containers and OVAs to get different setups than the real thing). Image-building code elsewhere has this:

````
      PKGARCH="`uname -m`"
      case "$PKGARCH" in
        x86_64) PKGARCH="amd64" ;;
        armv7l) PKGARCH="armhf" ;;
      esac
````

Also can look at product-dependent packages present in the build root, of which probably `bios-web-ipm-infra` vs `bios-web-ipm2` is the best functionality-related differentiator (there are also meta and trial pkgs that are more OVA vs non-OVA and so as bad a criteria as the CPU).

Reply:
script updated (if [ "`uname -m`" = x86_64 ];  ....)